pkgs/top-level/python-packages.nix

```diff
@@ -612,6 +612,7 @@ in {
   pyqt5 = pkgs.libsForQt5.callPackage ../development/python-modules/pyqt/5.x.nix {
     pythonPackages = self;
   };
+  pyqt5_with_qtwebkit = self.pyqt5.override { withWebKit = true; };
```

Comment:
Could you add a comment here explaining that this one should *not* be used by any Python libraries (so `pkgs/development/python-modules/*`) to avoid potential collisions.

Reply:
There can still be a collision on application level. Should I we move `pyqt5_with_qtwebkit` uses from `propagatedBuildInputs` to wrappers modifying `PYTHONPATH`? Or maybe use `python.withPackages`?